exampleFolder/index.js

```diff
@@ -0,0 +1,52 @@
+const express = require('express');
+var app = express();
+const axios = require('axios');
+const { response } = require('express');
+
+//POST Route (/addPerson)
+app.post('/addPerson', (req, res) => {
+    console.log(req.body)
+    const body = req.body
+
+    axios.post('http://java-sample-api-2020.herokuapp.com/addPerson',body)
+    .then((Response) => {
+        console.log(res.data)
+        res.status(200).json(res.data)
+    })
+
+    .catch((error) => {
+        console.log("Error Has Occured")
+        res.json({ErrorResponse: "Unsuccessful Route"})
+    })
+});
+
+//DELETE Route (/deletePerson/:id)
+app.delete('/deletePerson/:id', (req, res) => {
+    const { id } = req.params;
+
+    axios.delete('https://java-sample-api-2020.herokuapp.com/deletePerson?id='+id, id)
+    .then((Response) => {
+        console.log(res.data)
+        res.status(200).json(res.data)
```

Comment:
Assume 'Response' was already changed to 'response'. Here on line 30, inside the body of the .then method, 'res.data' does not return what I think you want.   'response.data' would be the variable you need.

Reply:
I think Jonathan is right here as well and I would have the same suggestion as before. Understanding what is happening at each line would be really beneficial.

Reply:
> Assume 'Response' was already changed to 'response'. Here on line 30, inside the body of the .then method, 'res.data' does not return what I think you want. 'response.data' would be the variable you need.

Fixed.